### DISCUSSION TITLE
Error while building the model server (Maven 3.9.5)

### DISCUSSION THREAD

**Discussion by Yunabell-VU** [eclipse-emfcloud/emfcloud#229] [ModelServer] [answered]
Hi all, 
I'm facing an issue with building the model server:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.2.0:check (default) on project org.eclipse.emfcloud.modelserver.parent: Failed during checkstyle execution: Unable to find suppressions file at location: .settings/checkstyle-suppressions.xml: Could not find resource '.settings/checkstyle-suppressions.xml'. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

The version of the model server I'm using is the latest with 139 commits.  
And my current maven is at version `3.9.5` (MacOs, M2 chip), but I also tried to use the maven wrapper with version `3.8.6` and `3.8.4` but still got the same problem. 

To use the wrapper alone leads to a successful build but there is no jar output in the target folder. 
```
mvn wrapper:wrapper -Dmaven=3.8.6
```
I can run the model server in Eclipse now, but I still need the standard alone JAR for my project. 

Thanks!

**Comment by ndoschek** [comment]
Hi @Yunabell-VU,
I will have a look and get back to you asap!
Cheers, Nina

**Comment by ndoschek** [comment, marked as answer]
Hi again @Yunabell-VU,
I just had a look and did a clean build on my system using Maven 3.9.2.
I did a fresh clone of the repository and could not reproduce your issue.

Could you please doublecheck if the file mentioned in the error message is availale on your environment? Perhaps it got deleted accidentally?
This is the file: https://github.com/eclipse-emfcloud/emfcloud-modelserver/blob/master/.settings/checkstyle-suppressions.xml

Cheers,
Nina

**Comment by Yunabell-VU** [comment]
Hi @ndoschek ,

Thanks for your reply. You're correct, I checked the files again and found that the whole `.settings` folder is removed for unknown reasons. After I copied everything inside, the build worked. Thanks a lot!

Cheers,
Yuna
